### ISSUE TITLE
Error when installing this gem on 4.0-stable

### ISSUE THREAD

**Issue by stydav** [AlchemyCMS/alchemy-dragonfly-s3#9]
Hello,

I have the following issue while migrating from file to s3 storage

```
gem 'alchemy_cms', github: 'AlchemyCMS/alchemy_cms', branch: '4.0-stable'
gem 'alchemy-dragonfly-s3', github: 'AlchemyCMS/alchemy-dragonfly-s3', branch: 'alchemy-4'
```

after adding the gem 'alchemy-dragonfly-s3' the following error appears:

`uninitialized constant Alchemy::BaseRecord`

Trace:

```
Showing /usr/local/bundle/bundler/gems/alchemy_cms-fd1a2312464a/app/views/alchemy/essences/_essence_picture_view.html.erb where line #5 raised:

uninitialized constant Alchemy::BaseRecord

app/views/alchemy/elements/_image_text_view.html.erb:9:in `block (2 levels) in _app_views_alchemy_elements__image_text_view_html_erb__1429976128264072401_70326751396960'
app/views/alchemy/elements/_image_text_view.html.erb:2:in `block in _app_views_alchemy_elements__image_text_view_html_erb__1429976128264072401_70326751396960'
app/views/alchemy/elements/_image_text_view.html.erb:1:in `_app_views_alchemy_elements__image_text_view_html_erb__1429976128264072401_70326751396960'
app/views/alchemy/page_layouts/_article.html.erb:21:in `_app_views_alchemy_page_layouts__article_html_erb__4039143447039554606_70326755549960'

```

**Comment by tvdeyen** [comment]
Solved by #11

**Comment by tvdeyen** [comment]
Until this has been merged you can use the branch in your app

```ruby
gem 'alchemy-dragonfly-s3', github: 'AlchemyCMS/alchemy-dragonfly-s3', branch: 'fix-missing-base-record'
```
